temboardagent/scripts/agent.py

```diff
@@ -88,6 +91,43 @@ def list_options_specs():
     yield OptionSpec(s, 'destination', default='/dev/log')
 
 
+class PluginManager(object):
+    def __init__(self, config=None):
+        self.config = config
+        self.plugins = {}
+
+    def fetch_plugins(self):
+        unloaded_names = filter(
+            lambda name: name not in self.config.plugins,
+            self.config.temboard.plugins
+        )
+        for name in unloaded_names:
+            logger.debug("Looking for plugin %s.", name)
+            for ep in iter_entry_points('temboardagent.plugins', name):
+                logger.info("Found plugin %s.", ep)
+                yield ep
+                self.config.plugins.pop(name, None)
+                break
+            else:
+                raise ConfigurationError("Missing plugin: %s." % (name,))
+
+    def load_plugins(self, entrypoints):
+        for ep in entrypoints:
+            try:
+                cls = ep.load()
+            except Exception:
+                logger.exception("Error while loading %s.", ep)
+                raise ConfigurationError("Failed to load %s." % (ep.name,))
+            else:
+                self.plugins[ep.name] = cls()
+
+
+def load_plugins(config):
+    manager = PluginManager(config)
+    manager.load_plugins(manager.fetch_plugins())
+    return manager
+
+
```

Comment:
Is it the right place for this class definition ?

Reply:
Je pensais le déplacer dans son propre module quand je remplacerai le code existant.

Pour le moment, je l'ai  met là car seul le script `agent` est extensible. Je peux le mettre dès maintenant dans `pluginsmgmt.py`, c'est pas un problème. Comme j'ai fait pour les deux codes de configuration dans `config.py`.

Dis-moi ce que t'en pense.